src/lib/UtapiClient.js

```diff
@@ -169,33 +167,26 @@ export default class UtapiClient {
      */
     _getParamsArr(params) {
         this._checkMetricTypes(params);
+        const props = [];
+        const { byteLength, newByteLength, oldByteLength, numberOfObjects } =
+            params;
+        const metricData = {
```

Comment:
you can probably just do `const metricData = Object.assign({}, params)` right? Is there an advantage to doing it this way?